addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -158,7 +158,10 @@ const FormEditor = options.Class.extend({
             field.id = generateHTMLId();
         }
         const template = document.createElement('template');
-        template.innerHTML = qweb.render("website.form_field_" + field.type, {field: field}).trim();
+        template.innerHTML = qweb.render(
+            "website.form_field_" + field.type,
+            {field: field, defaultName: _t("Field")}
```

Comment:
when key = value you can do this:
```suggestion
            {field, defaultName: _t("Field")}
```

Reply:
I agree with @qsm-odoo that it's clearer like this, so left it.